methods/sketch/src/pg_gp/sql/mfv.sql_in

```diff
@@ -22,3 +22,4207 @@ select mfvsketch_top_histogram(NULL::bytea,5) from generate_series(1,100);
+(1110,'M',0.52,0.4,0.145,0.7765,0.3525,0.1845,0.185,9),
+(1111,'I',0.52,0.38,0.135,0.5395,0.2295,0.133,0.157,8),
+(1112,'I',0.52,0.38,0.125,0.5545,0.288,0.1295,0.167,8),
+(1113,'F',0.52,0.46,0.15,1.019,0.523,0.1985,0.254,7),
+(1114,'I',0.525,0.4,0.13,0.6455,0.325,0.1245,0.17,8),
+(1115,'I',0.525,0.4,0.14,0.601,0.2625,0.1285,0.1835,9),
+(1116,'M',0.525,0.405,0.12,0.7555,0.3755,0.1555,0.201,9),
+(1117,'I',0.525,0.395,0.12,0.608,0.297,0.1395,0.1405,8),
+(1118,'I',0.53,0.4,0.125,0.617,0.279,0.127,0.19,8),
+(1119,'I',0.535,0.39,0.125,0.599,0.2595,0.149,0.169,9),
+(1120,'I',0.54,0.42,0.14,0.6665,0.3125,0.138,0.1895,10),
+(1121,'M',0.545,0.39,0.135,0.7835,0.4225,0.1815,0.156,7),
+(1122,'M',0.545,0.41,0.12,0.793,0.434,0.1405,0.19,9),
+(1123,'M',0.545,0.415,0.14,0.82,0.4615,0.127,0.218,9),
+(1124,'F',0.55,0.415,0.135,0.8145,0.427,0.1855,0.175,8),
+(1125,'F',0.55,0.43,0.15,0.84,0.395,0.195,0.223,8),
+(1126,'M',0.55,0.425,0.15,0.8315,0.411,0.1765,0.2165,10),
+(1127,'M',0.56,0.43,0.145,0.8995,0.464,0.1775,0.234,9),
+(1128,'M',0.56,0.445,0.16,0.8965,0.42,0.2175,0.2215,8),
+(1129,'F',0.56,0.44,0.155,0.6405,0.336,0.1765,0.245,8),
+(1130,'M',0.56,0.415,0.145,0.852,0.43,0.1885,0.205,8),
+(1131,'M',0.565,0.455,0.15,0.9595,0.4565,0.2395,0.23,9),
+(1132,'M',0.565,0.435,0.15,0.99,0.5795,0.1825,0.206,8),
+(1133,'F',0.565,0.45,0.175,1.0095,0.447,0.2375,0.2645,9),
+(1134,'M',0.57,0.46,0.15,1.0375,0.5415,0.2035,0.25,9),
+(1135,'F',0.57,0.445,0.145,0.8775,0.412,0.217,0.22,8),
+(1136,'I',0.57,0.44,0.15,0.755,0.3425,0.16,0.224,8),
+(1137,'F',0.575,0.46,0.145,0.9945,0.466,0.229,0.265,7),
+(1138,'F',0.575,0.45,0.16,1.068,0.556,0.214,0.2575,10),
+(1139,'M',0.575,0.435,0.14,0.8455,0.401,0.191,0.222,9),
+(1140,'F',0.575,0.47,0.165,0.869,0.435,0.197,0.238,9),
+(1141,'M',0.575,0.455,0.135,0.907,0.4245,0.197,0.26,9),
+(1142,'I',0.575,0.435,0.13,0.805,0.3155,0.2155,0.245,10),
+(1143,'M',0.575,0.445,0.17,1.0225,0.549,0.2175,0.228,9),
+(1144,'M',0.575,0.445,0.145,0.847,0.415,0.1945,0.22,9),
+(1145,'M',0.58,0.455,0.15,1.114,0.4765,0.2155,0.265,8),
+(1146,'M',0.58,0.455,0.195,1.859,0.945,0.426,0.441,9),
+(1147,'M',0.58,0.445,0.135,0.814,0.3775,0.1915,0.22,9),
+(1148,'M',0.58,0.45,0.14,0.9615,0.486,0.1815,0.253,9),
+(1149,'M',0.58,0.45,0.145,1.0025,0.547,0.1975,0.2295,8),
+(1150,'F',0.58,0.45,0.155,0.93,0.385,0.246,0.265,9),
+(1151,'M',0.585,0.46,0.145,0.9335,0.478,0.1825,0.235,9),
+(1152,'M',0.585,0.465,0.16,0.9555,0.4595,0.236,0.265,7),
+(1153,'M',0.59,0.47,0.15,0.9955,0.481,0.232,0.24,8),
+(1154,'F',0.6,0.475,0.16,1.0265,0.485,0.2495,0.2565,9),
+(1155,'M',0.6,0.455,0.17,1.1915,0.696,0.2395,0.24,8),
+(1156,'F',0.6,0.465,0.15,1.1025,0.5455,0.262,0.25,8),
+(1157,'M',0.6,0.465,0.155,1.0165,0.512,0.2465,0.225,10),
+(1158,'F',0.605,0.47,0.165,1.1775,0.611,0.2275,0.292,9),
+(1159,'M',0.605,0.475,0.14,1.1175,0.555,0.257,0.274,9),
+(1160,'M',0.605,0.48,0.17,1.1835,0.582,0.2365,0.317,10),
+(1161,'F',0.605,0.475,0.165,1.056,0.433,0.2195,0.357,9),
+(1162,'M',0.61,0.485,0.16,1.014
```

Comment:
Can we also add a test for `mfvsketch_top_histogram` ?

Reply:
This issue doesn't apply to `mfvsketch_top_histogram` since it does not have a merge function.